src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -533,10 +578,21 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 			if let hoursSinceHighRiskWarningAtTestRegistration = metadata.hoursSinceHighRiskWarningAtTestRegistration {
 				$0.hoursSinceHighRiskWarningAtTestRegistration = hoursSinceHighRiskWarningAtTestRegistration
 			}
+			if let daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration = metadata.daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration {
+				$0.ptDaysSinceMostRecentDateAtRiskLevelAtTestRegistration = daysSinceMostRecentDateAtCheckinRiskLevelAtTestRegistration
+			}
+			if let hoursSinceCheckinHighRiskWarningAtTestRegistration = metadata.hoursSinceCheckinHighRiskWarningAtTestRegistration {
+				$0.ptHoursSinceHighRiskWarningAtTestRegistration = hoursSinceCheckinHighRiskWarningAtTestRegistration
+			}
+			// special handling for the triStateBool
+			if let submittedWithCheckins = metadata.submittedWithCheckIns {
+				$0.submittedWithCheckIns = submittedWithCheckins ? .tsbTrue : .tsbFalse
+			} else {
+				$0.submittedWithCheckIns = .tsbFalse
+			}
 			if let submittedWithTeleTan = metadata.submittedWithTeleTAN {
 				$0.submittedWithTeleTan = submittedWithTeleTan
 			}
-
 			$0.submittedAfterRapidAntigenTest = metadata.submittedAfterRapidAntigenTest
```

Comment:
if let submittedAfterRapidAntigenTest = metadata.submittedAfterRapidAntigenTest {
                  $0.submittedAfterRapidAntigenTest = submittedAfterRapidAntigenTest
}

Reply:
No, because it is a bool and not optional, we do not need that here.